### ISSUE TITLE
Pipe connects to EIO facades

### ISSUE THREAD

**Issue by TheQuadShot** [micdoodle8/Galacticraft#3342]
Minecraft 1.10.2
Forge 12.18.3.2511
Galacticraft-Planets-1.10.2-4.0.1.173
GalacticraftCore-1.10.2-4.0.1.173
MicdoodleCore-1.10.2-4.0.1.173
modlist https://pastebin.com/0ebnhFge

The image shows a valid connection to an ender tank(top of the "t" on the right) and the lower is a facade painted w/clear glass covering a power conduit, the one on the top is just placed there next to a normal stone brick.

https://imgur.com/a/mEvaD

**Comment by radfast** [comment]
Could be their bug not ours.  Would you be able to check whether pipes from other mods (i.e. not from EnderIO itself) show the same behaviour with facades?

**Comment by TheQuadShot** [comment]
Yup, I'll file this over there, I got Extra Utils to connect but not Embers those are the only other pipes I have in this "pack".

**Comment by TheQuadShot** [comment]
evidently it's not theirs. Reopening
